ios/Classes/BetterPlayer.m

```diff
@@ -239,9 +239,9 @@ - (void)setDataSourcePlayerItem:(AVPlayerItem*)item withKey:(NSString*)key{
     [_player replaceCurrentItemWithPlayerItem:item];
 
     if (!@available(iOS 16.0, *)) {
-         // set buffer time into 5 seconds
+         // set buffer time into 1 seconds
```

Comment:
I think now buffering period become **minimum** 1 second, but not strictly 1 second. Isn't it?
Please check previous comment: https://github.com/dwango-nfc/betterplayer/pull/8#discussion_r1138365157

If so, can you change this comment to describe nicely?